src/explore-education-statistics-admin/src/components/form/FormFieldEditor.tsx

```diff
@@ -162,11 +162,12 @@ function getInvalidLinks(elements: Element[]) {
       const url = attributes.linkHref as string;
 
       try {
-        // exclude anchor links and localhost as they fail Yup url validation.
+        // exclude anchor links, localhost and emails as they fail Yup url validation.
         if (
           url &&
           !url.startsWith('#') &&
-          !url.startsWith('http://localhost')
+          !url.startsWith('http://localhost') &&
+          !url.startsWith('mailto:')
```

Comment:
I also wondered if changes need to be made to the html sanitiser?  Can imagine it stripping out the `mailto`.

Reply:
The sanitizer allows `mailto` so should be fine.